src/EventStore.Core/MetricsBootstrapper.cs

```diff
@@ -84,15 +82,15 @@ public static class MetricsBootstrapper {
 		if (conf.Events.TryGetValue(Conf.EventTracker.Read, out var readEnabled) && readEnabled) {
 			var readTag = new KeyValuePair<string, object>("activity", "read");
 			trackers.TransactionFileTracker = new TFChunkTracker(
-				readBytes: new CounterSubMetric<long>(byteMetric, readTag),
-				readEvents: new CounterSubMetric<long>(eventMetric, readTag));
+				readBytes: new FastCounterMetric(coreMeter, "eventstore-io", readTag,  "bytes"),
+				readEvents: new FastCounterMetric(coreMeter, "eventstore-io", readTag, "events"));
```

Comment:
im slightly suspicious of creating two separate instruments that output to the same name (`eventstore-io-events`), unless you saw some doc that indicates that this is definitely ok? otherwise i expect we need a `CounterMetric` class and a `CounterSubMetric` class

Reply:
Functionally, this won't cause any issues. `coreMeter` will have two observable counters with name `eventstore-io-events` and both counters will correctly report `read` and `write` tagged counts respectively. But, I think having submetric might help in code readability.

Reply:
I have changed the implementation.